obspy/core/inventory/network.py

```diff
@@ -407,13 +439,23 @@ def select(self, station=None, location=None, channel=None, time=None,
                 if not sta.is_active(time=time, starttime=starttime,
                                      endtime=endtime):
                     continue
+            geo_filters = (minlatitude, maxlatitude,
```

Comment:
I knoooow I'm a pit picky on that one now, but didn't you say it's not a filter ? ;)

Reply:
just a local variable, so doesnt really matter